docs: admin: Add process mgmt, project files, security and resources (#104)
OpenDataServices/standards-lab: docs/admin/project-files.md

# Project Files

> Projects are stored in a directory that is specified by setting `ROOT_PROJECTS_DIR` in the django settings. The default is `/tmp/standards-lab`.

Reviewer: ```suggestion
Projects are stored in a directory that is specified by setting `ROOT_PROJECTS_DIR` in the Django settings. The default is `/tmp/standards-lab`.
```

> The value of `ROOT_PROJECTS_DIR` must be created before using Standards Lab.

Reviewer: ```suggestion
The `ROOT_PROJECTS_DIR` directory must be created before using Standards Lab.
```

> The project directory includes, uploaded data, uploaded schema and the project's current settings.

Reviewer: ```suggestion
The project directory includes: uploaded data, uploaded schema, and the project's current settings.
```
